README.md

```diff
@@ -53,6 +53,123 @@ See [RADIUS.md](RADIUS.md) for a list of supported Attrbiutes.
 
 Many thanks to [On Waves](https://www.on-waves.com/) for sponsoring the RADIUS Authentication implementation.
 
+Example of configuration **RADIUS**:
+```erlang
+%% ...
+{ergw_aaa, [
+    {handlers, [
+        {ergw_aaa_static, [
+            {'Node-Id',        <<"CHANGE-ME">>},            %% <- CHANGE
+            {'NAS-Identifier', <<"CHANGE-ME">>},            %% <- CHANGE
+            {'NAS-IP-Address', {127,0,0,3}},                %% <- CHANGE
+            {'Acct-Interim-Interval',   1800},              %% <- CHANGE
+            {'Framed-Protocol',         'PPP'},
+            {'Service-Type',            'Framed-User'}
+        ]},
+        {ergw_aaa_radius, [
+            {server,
+                {{127,0,0,4}, 1813, <<"CHANGE-ME-SECRET">>} %% <- CHANGE IP and SECRET
+            }
+        ]}
+    ]},
+    {services, [
+        {'Default', [
+            {handler, 'ergw_aaa_static'}
+        ]},
+        {'RADIUS-Acct', [
+            {handler, 'ergw_aaa_radius'}
+        ]}
+    ]},
+    {apps, [
+        {default, [
+            {session, ['Default']},
+            {procedures, [
+                {authenticate, []},
+                {authorize, []},
+                {start, ['RADIUS-Acct']},
+                {interim, ['RADIUS-Acct']},
+                {stop, ['RADIUS-Acct']},
+                {{gx, 'CCR-Initial'},   []},
+                {{gx, 'CCR-Update'},    []},
+                {{gx, 'CCR-Terminate'}, []}
+            ]}
+        ]}
+    ]}
+]},
+%% ...
+```
+Example of configuration **epc-ocs** `function` of **DIAMETER**:
+```erlang
+%% ...
+{ergw_aaa, [
+%% ...
+    {functions, [
+        {'epc-ocs', [
+            {handler, ergw_aaa_diameter},
+            {'Origin-Host', <<"CHANGE-ME">>},                           %% <- CHANGE
```

Comment:
the Origin-Host needs to be resolvable to local IP (either through /etc/hosts or DNS).

As alternative to having a resolvable name, it can also be specified as `{Host :: binary(), IP :: inet:ip_address()}`

Reply:
@RoadRunnr but why is the origin-host entity needs to be resolvable? it is its own "id" … the transport-definition later takes care about the "here is where to connect to" and the entity connected to is able to transport messages to the "id" referenced in here.

i totally get the {connect_to, ...} part of the transport section and that the diameter stack needs to be able to resolve the target. but origin-host?

Reply:
Origin-Host (and Origin-Realm) are use by DIAMETER relays and proxies to route the requests. They are completely permitted to attempt to establish new (not predefined) connections for that. Therefore the name needs to resolvable.

Also, that is how the Erlang DIAMETER stack works. If you don't specify a local IP to bind, it tries to use the Origin-Host for that.